### DISCUSSION TITLE
Android - Warpcast error unable to create account even fee is paid

### DISCUSSION THREAD

**Discussion by ithmz** [farcasterxyz/protocol#157] [Q&A] [answered]
Dev please take a look at this issue. I paid for annually fee through Google Play but it keeps showing this behavior. Can you guys come check at it. I can include the Google Play bill if needed for extra investigation

![431117212_923635425925295_5381268476335323616_n](https://github.com/farcasterxyz/protocol/assets/23276580/8c9169ee-5eb4-4da1-ada0-b00422ace942)

**Comment by horsefacts** [comment, marked as answer]
Hi there, for Warpcast client support you can email [support@merklemanufactory.com](mailto:support@merklemanufactory.com).

This discussion forum is for the Farcaster protocol, so I'm going to close this conversation. Please reach out to the support email above if you need help with Warpcast.

**Reply by ithmz** [reply]
ok you can close it now since the Google Play refunded the full amount
